app/lib/continuousTimeUtils.js

```diff
@@ -28,7 +28,7 @@ function addTimePadding(parsedTimes, isOneSession) {
     if (time.sessions.length < max) {
       // eslint-disable-next-line no-param-reassign
       time.padding = (max - time.sessions.length);
-    } else if ((time.sessions.length === 1) && (isOneSession === true)) {
+    } else if ((time.sessions.length === 1) || (isOneSession === true)) {
```

Comment:
Well spotted. I set it to `||` as it wasn't adding the `one-session` class for "normal" opening hours. I've made that work by setting the flag to true in [`openingTimesMapper.js`](https://github.com/nhsuk/profiles/pull/151/commits/cec41ce6a225501377e05f017653c321d4603818#diff-8e199386d5cabc6984ff1b2c134f9236)